src/systemtest/java/com/ericsson/ei/systemtest/utils/StepsUtils.java

```diff
@@ -169,6 +174,53 @@ public static void deleteSubscriptions(String frontendBaseUrl, String backendBas
         }
     }
 
+    /**
+     * Triggers a jenkins job with parameters.
+     *
+     * @param jenkinsJobToTrigger
+     * @param jenkinsToken
+     * @throws Exception
+     */
+    public static void triggerJenkinsJobWithParameters(String jenkinsJobToTrigger, Map<String, String> parameters, String jenkinsToken) throws Exception {
+        boolean success = jenkinsManager.buildJobWithParameters(jenkinsJobToTrigger, jenkinsToken, parameters);
+        assertEquals("Was not able to trigger jenkins job", true, success);
+    }
+
+    /**
+     * Check if all jenkins jobs in a list of jenkins job names has been triggered
+     *
+     * @param jenkinsJobNames
+     * @throws Exception
+     */
+    public static void hasJenkinsJobsBeenTriggered(ArrayList<String> jenkinsJobNames) throws Exception {
+        for(int i=0; i<jenkinsJobNames.size(); i++) {
+            hasJenkinsJobBeenTriggered(jenkinsJobNames.get(i));
+        }
+    }
+
+    /**
+     * Check if a single jenkins job has been triggered
+     *
+     * @param jenkinsJob
+     * @throws Exception
+     */
+    public static void hasJenkinsJobBeenTriggered(String jenkinsJob) throws Exception {
+        long maxTime = System.currentTimeMillis() + 60000;
+
+        while(System.currentTimeMillis() < maxTime) {
+            try {
+                JSONObject status = jenkinsManager.getJenkinsBuildStatusData(jenkinsJob);
+                LOGGER.info(jenkinsJob + " was triggered.");
```

Comment:
Doesn't status include some relevant status info that can be valuable for the logs.